services/user.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str | None = None,
+    first_name: str | None = None,
+    last_name: str | None = None,
+) -> User:
+    kwargs = {}
+
+    if email:
+        kwargs["email"] = email
+
+    if first_name:
+        kwargs["first_name"] = first_name
+
+    if last_name:
+        kwargs["last_name"] = last_name
+
+    return get_user_model().objects.create_user(
+        username,
+        password=password,
+        **kwargs,
+    )
```

Comment:
Looks like in this case there's no need to create kwargs dictionary, what do you think?